src/push/monitor.ts

```diff
@@ -254,17 +258,36 @@ export function buildMonitorFromYaml(
   return mon;
 }
 
-export const parseAlertConfig = (config: MonitorConfig) => {
+export const parseAlertConfig = (
+  config: MonitorConfig,
+  globalAlertConfig?: AlertConfig
+) => {
+  const alertConfig = {};
   if (config['alert.status.enabled'] !== undefined) {
     const value = config['alert.status.enabled'];
     delete config['alert.status.enabled'];
+    alertConfig['status'] = {
+      enabled: value,
+    };
+  }
+  if (config['alert.tls.enabled'] !== undefined) {
+    const value = config['alert.tls.enabled'];
+    delete config['alert.tls.enabled'];
+    alertConfig['tls'] = {
+      enabled: value,
+    };
+  }
+  const monitorAlertConfig =
+    Object.keys(alertConfig).length > 0 ? alertConfig : config.alert;
```

Comment:
not sure why this is required, wouldnt it be safer to just use do this
```
return {...globalConfig, ...alertConfig }
```

`config.alert` if present would fill in `alertConfig` anyways?

Reply:
done !!